alpha/apps/kaltura/lib/kUploadTokenMgr.php

```diff
@@ -341,10 +341,17 @@ protected function handleResume($fileData, $resumeAt)
 		return $currentFileSize;
 	}
 	
-	private function appendCurrentChunk($targetFilePath, $chunkFilePath)
+	private function appendCurrentChunk($targetFilePath, $chunkFilePath, $resumeAt)
 	{
 		$targetFileResource = fopen($targetFilePath, 'r+b');
-		fseek($targetFileResource, 0, SEEK_END);
+		if($resumeAt == -1)
```

Comment:
It is better to use a const/defined value that will be used as end of file markup.